examples/blog-with-comment/pages/api/comment.js

```diff
@@ -0,0 +1,109 @@
+import { nanoid } from 'nanoid'
+import redis from '../../lib/redis'
+
+async function fetchUser(token) {
+  const response = await fetch(
+    `https://${process.env.NEXT_PUBLIC_AUTH0_DOMAIN}/userinfo`,
+    {
+      headers: {
+        Authorization: `Bearer ${token}`,
+        'Content-Type': 'application/json',
+      },
+    }
+  )
+  return await response.json()
+}
+
+export default async function handler(req, res) {
+  // FETCH COMMENTS
```

Comment:
Suggestion: rather than a comment, extract this to a function `fetchComments()` along with the other chunks of logic here, into a `lib` file. This would make this API route read more straight forward - could even use a switch statement for the `req.method`

Reply:
Very good suggestion. I applied it 👍